requirements-test.txt

```diff
@@ -1,2 +1,2 @@
-coverage
-coveralls
+coverage==6.5.0
+coveralls==3.3.1
```

Comment:
There's a later version of coverage available, but 6.5.0 was the latest supported version for coveralls 3.3.1.